eos-core-depends

```diff
@@ -101,6 +101,7 @@ fonts-vlgothic
 fonts-wqy-zenhei
 foomatic-db-compressed-ppds
 fprintd
+fuse-overlayfs
```

Comment:
I think you want `# For rootless podman` here?

Reply:
Done!